FrenchFilter/sections/adservers.txt

```diff
@@ -1,8 +1,9 @@
 !
 ! Section contains list of advertising networks
 !
+||dustymural.com^$third-party
 ||mystighty.info^
 ||i4tvy8vk9f74.com^$third-party
 ||sohjah-thahka.info^
 ||goitcl.com^
-||mineralisasse.site^$third-party
\ No newline at end of file
+||mineralisasse.site^$third-party

```

Comment:
I don't see it

Reply:
I didn't add that. It went down after adding to ad server.